### ISSUE TITLE
Thirdweb Latest Registry Addresses and Factory Addresses 

### ISSUE THREAD

**Issue by iggyiccy** [thirdweb-dev/contracts#600]
Hi There, 

Would like someone to help pointing me to the right documentation if possible. I have been updating my code base and use to use the following address for Thirdweb registry and interacting with the factory. 

But recently the documentation https://portal.thirdweb.com/contracts/TWRegistry seems to be removed. Are someone able to let me know if the following registry address and factory address are the latest one the Thirdweb dashboard frontend been using? 

```
twRegistryAddress ="0x7c487845f98938Bb955B1D5AD069d9a30e4131fd"
twFactoryAddress ="0x76F948E5F13B9A84A81E5681df8682BBf524805E"
ozDefenderForwarderAddress ="0xc82BbE41f2cF04e3a8efA18F7032BDD7f6d98a81"
```

I am pretty sure the Open Zeppelin Forwarder Address is right - https://github.com/thirdweb-dev/ozdefender-autotask

**Comment by warengonzaga** [comment]
Hello @iggyiccy, are you looking to confirm that these addresses are up to date right?

**Comment by warengonzaga** [comment]
Hello @kumaryash90 @nkrishang, could you please check the address to see if this is correct? Thanks!

**Comment by iggyiccy** [comment]
> Hello @iggyiccy, are you looking to confirm that these addresses are up to date right?

Yes, I wish to confirm these addresses are up-to-date! Thanks in advance 😃

**Comment by iggyiccy** [comment]
> hey @iggyiccy how do you plan to use these addresses?

Hi Kumar, 

We have been using an fork of the [Thirdweb Go SDK](https://github.com/thirdweb-dev/go-sdk) for our Golang backend API cause the community one wasn't being maintain since Jul 2022. 

These addresses are part of the constants here: 
[https://github.com/qnfnypen/thirdweb-go-sdk/blob/ef0793878921018db5ff3ca4a460c2a88ef5ed7a/thirdweb/constants.go#L198](https://github.com/qnfnypen/thirdweb-go-sdk/blob/ef0793878921018db5ff3ca4a460c2a88ef5ed7a/thirdweb/constants.go#L198)

**Comment by iggyiccy** [comment]
I am thinking are these addresses on the typescript sdk the right one? But they are not updated the last 8 months. Anyone able to confirm? 

https://github.com/thirdweb-dev/js/blob/771bd1e1b280681992195103c22aafa1fb98f7e0/packages/sdk/src/evm/constants/addresses/CONTRACT_ADDRESSES.ts#L5

**Comment by warengonzaga** [comment]
Hello @iggyiccy, I notified the engineering team internally. I'm sorry for the delay.

**Comment by nkrishang** [comment]
@iggyiccy The addresses you've posted in the original comment are the latest ones: https://github.com/thirdweb-dev/contracts/issues/600#issue-2097111146

Particularly, `0x76F948E5F13B9A84A81E5681df8682BBf524805E` is a [TWCloneFactory](https://etherscan.io/address/0x76F948E5F13B9A84A81E5681df8682BBf524805E#code) contract with only a `deployProxyByImplementation` function, whereas the address in the SDK `0x5DBC7B840baa9daBcBe9D2492E45D7244B54A2A0` is a [TWFactory](https://etherscan.io/address/0x5DBC7B840baa9daBcBe9D2492E45D7244B54A2A0#code) contract which contains additionall logic that we don't use anymore.
